.github/workflows/ci.yml

```diff
@@ -51,4 +61,4 @@ jobs:
 
       - name: Run pipeline with test data
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test_github,singularity --outdir ./results
+          nextflow run ${GITHUB_WORKSPACE} -profile test_github,singularity -c conf/test_github_params.config --outdir ./results
```

Comment:
Can we merge 2 test_github config files together?

Reply:
The function of having `conf/test_github_params.config ` is to override HIFIASM and OATK settings from the ones set in the modules.conf file. I wouldn't work by setting them in the config corresponding to the test_github profile. Do you know how to force overriding the ext.args in the profile config?

Reply:
If it works in 2 config files, there is no reason for it not working in one file.

You can see the the extra setting for **oatk** in the one config being picked up.
```
Command executed:

  oatk \
      -k1001 -c90 -Ttmp \
      -m insecta_mito.fam \
       \
      -t 2 \
      -o baUndUnlc1 \
      baUndUnlc1.fasta
```

The actual error message is 
`.command.run: line 24: /usr/bin/awk: No such file or directory`

If you go to the container, you can see the `awk` being installed here: `/usr/local/bin/oatk`

Reply:
@gq1 It failed in the first place because it didn't pick up the right setting: it must have been -c5 from `conf/test_github.config` instead of -c 90 from `conf/modules.config` 
```
[E::pathfinder] no organelle component found
[E::main] pathfinder program failed
```

Reply:
Sorry I didn't see the details.

I tried it on the farm and have the same problem. However, I didn't get this error: `/usr/bin/awk: No such file or directory`

Maybe @muffato knows, can we overwrite the process's `ext.args` in the profile config file, which is already defined  in the modules.config?

Reply:
Yep ! I had to do that in the genome-note pipeline to add some BUSCO options in test mode:
https://github.com/sanger-tol/genomenote/blob/1.1.1/conf/modules.config#L52-L59

Reply:
`modules.config` is loaded after profile config file so it will overwrite the setting there.

Reply:
It seems working now.